lib/dns/src/opstrace.ts

```diff
@@ -158,10 +158,11 @@ export class DNSClient {
         clustername
       }
     };
-    return axios.request(deleteRequest);
```

Comment:
Here we _did not_ wait for the response, but moved on with the program.

Likewise any corresponding axios error corresponding with this dangling task would be thrown in the context of things that are difficult see through.

Potentially, we didn't even wait for the HTTP response to arrive before terminating the program  -- possible, when DNS deletion happens towards the end of the `destroy` operation`. Which it does:

https://github.com/opstrace/opstrace/blob/eb32d8ef4e563d9964388a82d8fc9bb8c187eac9/packages/uninstaller/src/gcp.ts#L108

This code change makes it _much_ easier to reason about where/when an unexpected HTTP response (non-2xx in this case I guess, axios default) is expected to throw an error.